packages/compute-baseline/README.md

```diff
@@ -40,6 +38,33 @@ Run:
 
 ## Usage
 
+### Get a Baseline status for a portion of a feature
+
+```javascript
+import { getStatus } from "compute-baseline";
+
+getStatus("fetch", "api.Response.json");
+```
+
+Returns:
+
+```
+{
+  baseline: 'high',
+  baseline_low_date: '2017-03-27',
+  baseline_high_date: '2019-09-27',
+  support: {
+    chrome: '42',
+    chrome_android: '42',
+    edge: '14',
+    firefox: '39',
+    firefox_android: '39',
+    safari: '10.1',
+    safari_ios: '10.3'
+  }
+}
+```
+
 ### Check support for a group of compat keys
 
```

Comment:
```suggestion

**Note**: the code shown below returns support data that has not received an editorial review. Do not use for presenting a Baseline status. See [Limitations](#limitations).

```

Reply:
I think it would be safe to add an intro like this here, as well as below in "Check support for a single support key" and in "Bring your own compatibility data".

Reply:
Good idea. Done.